app/jobs/patient_list_download_job.rb

```diff
@@ -1,14 +1,21 @@
-class PatientListDownloadJob < ApplicationJob
+class PatientListDownloadJob
+  include Sidekiq::Worker
+
+  sidekiq_options lock: :until_executed,
+    lock_timeout: 15.minutes,
+    lock_ttl: 15.minutes,
+    on_server_conflict: :reject
+
   def perform(recipient_email, model_type, params)
     case model_type
     when "facility"
-      model = Facility.find(params[:facility_id])
-      model_name = model.name
-    when "facility_group"
-      model = FacilityGroup.find(params[:id])
+      model = Facility.find(params["facility_id"])
```

Comment:
Is this hash not indifferent-access?

Reply:
No, not sure why but `Sidekiq::Worker`'s `perform_later` doesn't pass indifferent hashes.